docs/docs/cypress.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Cypress"
```

Comment:
Sure, that’s an option. It just feels a bit „opinionated“ as I chose these libaries because I like them best but there are many others out there.

I thought that the libaries itself would be grouped (with the accordion) under the categories you named.